File: CS/13_Conversion/SVGToPDF.cs
using Spire.Pdf;
using Spire.Pdf.Annotations;
using Spire.Pdf.Graphics;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace SVGToPDF
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a new PDF document.
            PdfDocument doc = new PdfDocument();

            //Load the file from disk.
            doc.LoadFromSvg(@"..\..\..\..\..\..\Data\template.svg");

            //Save the document
            String result = "SVgToPDF_out.pdf";
            doc.SaveToFile(result);

            //Launch the Pdf file
            PDFDocumentViewer(result);
        }

        private void PDFDocumentViewer(string filename)
        {
            try
            {
                System.Diagnostics.Process.Start(filename);
            }
            catch { }
        }
    }
}

File: VB/13_Conversion/SVGToPDF.vb
Imports Spire.Pdf

Namespace SVGToPDF
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new instance of PdfDocument class
			Dim doc As New PdfDocument()

			' Load the SVG file from the specified path
			doc.LoadFromSvg("..\..\..\..\..\..\Data\template.svg")

			' Specify the output file name for the resulting PDF
			Dim result As String = "SVgToPDF_out.pdf"

			' Save the document as a PDF file
			doc.SaveToFile(result)

			' Close the PdfDocument object
			doc.Close()

			' Launch the Pdf file
			PDFDocumentViewer(result)
		End Sub

		Private Sub PDFDocumentViewer(ByVal filename As String)
			Try
				Process.Start(filename)
			Catch
			End Try
		End Sub
	End Class
End Namespace
